pkg/ifaces/watcher.go

```diff
@@ -191,9 +195,15 @@ func (w *Watcher) netnsNotify(ctx context.Context, out chan Event) {
 				}
 				if event.Op&fsnotify.Create == fsnotify.Create {
 					ns := filepath.Base(event.Name)
-					log.WithField("netns", ns).Debug("netns notification")
+					log.WithField("netns", ns).Debug("netns create notification")
+					w.nsDone[ns] = make(chan struct{})
```

Comment:
when creating a new channel for map key `ns`, is there any risk of collision / leak if there was already previously a channel stored in `w.nsDone[ns]` ? Might be safer to first check if it isn't empty, close the old channel ?

Reply:
I think that will be an issue as we suppose to know about netns creation or deletion once